wd_core.au3

```diff
@@ -1256,25 +1261,25 @@ Func _WD_Startup()
 			$sWinHttpVer &= " (Download latest source at <https://raw.githubusercontent.com/dragana-r/autoit-winhttp/master/WinHttp.au3>)"
 		EndIf
 
-		_WinAPI_GetBinaryType($_WD_DRIVER)
+		_WinAPI_GetBinaryType(@ScriptDir & "\" & $_WD_DRIVER)

```

Comment:
This change shouldn't occur as we removed this earlier today

Reply:
Ye... I gett v7 without fixes....

Reply:
Take a look on: [small fixes](https://github.com/Danp2/au3WebDriver/pull/242/commits/070aec7f1e04fa8069fdb99340a198018125a5ac)